react/features/user-customization/functions.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+/**
+ * Extracts the fqn part from a path.
+ *
+ * @param {string} path - The URL path.
+ * @returns {string}
+ */
+export function extractFqnFromPath(path: string) {
+    const parts = path.split('/');
+    const len = parts.length;
+
+    return parts.length > 2 ? `${parts[len - 2]}/${parts[len - 1]}` : '';
```

Comment:
This looks odd. When using a URL like `https://meet.jit.si/saghul` the pathname will be `/saghul` and if using a tenant like `https://meet.jit.si/lalala/saghul` it will be `/lalala/saghul`. Neither of them will result in having more than 2 slashes.

So either this is wrong or I don't understand what this FQN is supposed to be (the fact that it's not even explained in a comment doesn't really help). I assumed we want to send the tenant + room name.

Reply:
Yes, you're right. We want `tenant`/`roomName`. I have added that in the comment.

What you missed is the `split` part. So 
```'/'.split('/') is ['', ''].```

We check here only for urls which have tenant. If a meeting does not have a tenant we display the defaults.

More than that, there could be instances where the url looks like `domain.com/aaaa/tenant/roomName` so we cover those too.